docker/release/dockercomposefiles/docker-compose-2.x.yml

```diff
@@ -2,40 +2,40 @@
 version: '3'
 services:
   opensearch-node1:
-    image: opensearchproject/opensearch:2
+    image: opensearchproject/opensearch:latest
     container_name: opensearch-node1
     environment:
       - cluster.name=opensearch-cluster
       - node.name=opensearch-node1
       - discovery.seed_hosts=opensearch-node1,opensearch-node2
       - cluster.initial_cluster_manager_nodes=opensearch-node1,opensearch-node2
-      - bootstrap.memory_lock=true  # along with the memlock settings below, disables swapping
-      - OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m    # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
+      - bootstrap.memory_lock=true # along with the memlock settings below, disables swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
       - OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_INITIAL_ADMIN_PASSWORD}    # Sets the demo admin user password when using demo configuration, required for OpenSearch 2.12 and higher
```

Comment:
Do we need double quotes here? coming from https://github.com/opensearch-project/opensearch-build/issues/4115#issuecomment-1953322664.

Reply:
This is to keep consistent with the changes @AMoo-Miki and I doing on the website version.
The website version has been using double quotes the whole time.
Just to be consistent right here.

Reply:
Thanks, just tested `OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_INITIAL_ADMIN_PASSWORD} ` works on my local.

Reply:
Tweaked to use the original setup as the download page one have yaml linting issues.